requirements/requirements-test.txt

```diff
@@ -21,4 +21,5 @@ flash_attn==2.0.5
 datasets
 pydantic
 ray
+ordered_set
```

Comment:
Ordered set has been added in the third line.

Reply:
fixed